### PR TITLE
Fix grants of nfalco for dtkit plugin

### PR DESCRIPTION
**Pull request by nfalco79**
# Description

Discussion at https://groups.google.com/forum/#!topic/jenkinsci-dev/251sqn85_jM
@batmat This should fix the issue with the PR #1058 (or at least is what I hope)

Github repository of plugins/components

* https://github.com/jenkinsci/dtkit-plugin/
* https://github.com/jenkinsci/libdtkit/

### Always

- [x] Add link to plugin/component Git repository in description above

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by nfalco79** [comment]
@batmat could you have a look if this can fix my release issue?

**Comment by nfalco79** [comment]
@daniel-beck someone can help here?

**Comment by batmat** [comment]
Please, as usual, do **not** blank out the template. Looking.

**Comment by nfalco79** [comment]
> Please, as usual, do **not** blank out the template. Looking.

Fixed, I had see other PRs where if not a new access request the template seems could be skipped. Anyway I had add the same header of my previous PR

File: permissions/plugin-dtkit-api.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: "dtkit-api"
-github: "jenkinsci/dtkit-libdtkit"
+github: "jenkinsci/dtkit-plugin"
 paths:
 - "org/jenkins-ci/plugins/dtkit-api"
```
Comment (batmat) [inline permissions/plugin-dtkit-api.yml]: Given now IIUC you've got only one artifact, this ought to be enough?

**Comment by batmat** [comment]
The fact you restored the template did help me realize/remember you have actually two repositories.
https://github.com/jenkinsci/libdtkit/ does have that `...-frmk` artifact you'll need described, as I wrote before.

Please double-check each `groupId`/`artifactId` of all single modules in both your repositories, and compare to other multi-modules repositories (like blue-ocean e.g.)

**Comment by nfalco79** [comment]
The https://github.com/jenkinsci/dtkit-plugin already point to the released (only locally) build of libdtkit component.

**Comment by nfalco79** [comment]
Well. The library was composed into 3 parts.
A common library (composed by 3 artifacts) in https://github.com/jenkinsci/libdtkit/
Two plugins:
* xunit plugin https://github.com/jenkinsci/xunit-plugin
* dtkit plugin https://github.com/jenkinsci/dtkit-plugin/

Now the dtkit plugin has been parked and will be a API plugin like Jesse Glick heavy sugget.
So the xunit plugin instead to embed the dtkit component library can use the API plugin like jackson2 and others.

The dtkit library (component-dtkit-lib.yml) is composed by 4 maven project (one parent and 3 children) ready to be released:
* org.jenkins-ci.lib.dtkit/dtkit-frmk
* org.jenkins-ci.lib.dtkit/dtkit-metrics-util
* org.jenkins-ci.lib.dtkit/dtkit-metrics-model
* org.jenkins-ci.lib.dtkit/dtkit-metrics-hudson-api

These artifacts are written like paths in the component-dtkit-lib.yml as documented in the README.MD of this repository

The dtkit plugin (plugin-dtkit-api.yml) is composed just by a **single** maven project. This API plugin have GAV:
* org.jenkins-ci.plugins/dtkit-api

File: permissions/plugin-dtkit-api.yml (hunk 1 above)
Comment (nfalco79) [reply permissions/plugin-dtkit-api.yml]: plugin-dtkit-api.yml refer to the https://github.com/jenkinsci/dtkit-plugin repository that contains just only one plugin artifact.

**Comment by slide** [comment]
@daniel-beck @batmat requested your review on this, can you take a look?

**Comment by nfalco79** [comment]
@oleg-nenashev I have change the PR as requested. Every maven module/project is now into a single yml file.

**Comment by nfalco79** [comment]
news?

**Comment by oleg-nenashev** [comment]
It;s guess @batmat is busy with other things. Let;s merge as is and follow-up on any requested changes in subsequent PRs

**Comment by nfalco79** [comment]
Thanks very much. Today I will try to release the component libraries. Cross fingers!

**Pull request merged**